extensions/git/src/commands.ts

```diff
@@ -1890,6 +1890,48 @@ export class CommandCenter {
 		await repository.deleteTag(choice.label);
 	}
 
+	@command('git.deleteRemoteTag', { repository: true })
+	async deleteRemoteTag(repository: Repository): Promise<void> {
+		const picks = repository.refs.filter(ref => ref.type === RefType.Tag)
+			.map(ref => new TagItem(ref));
+
+		if (picks.length === 0) {
+			window.showWarningMessage(localize('no tags', "This repository has no tags."));
+			return;
+		}
+
+		const placeHolder = localize('select a tag to delete from remote', 'Select a tag to delete from remote');
+		const tag = await window.showQuickPick(picks, { placeHolder });
+
+		if (!tag) {
+			return;
+		}
+
+		const remotes = repository.remotes;
+
+		if (remotes.length === 0) {
+			window.showErrorMessage(localize('no remotes added', "Your repository has no remotes."));
+			return;
+		}
+
+		const picksRemote = remotes.filter(r => r.fetchUrl !== undefined).map(r => ({ label: r.name, description: r.fetchUrl! }));
+
+		if (picksRemote.length === 0) {
+			window.showWarningMessage(localize('no remotes', "This repository has no remotes."));
```

Comment:
This should be an error message, not warning. Also, the validation should happen before asking for user input.